src/sdk/pynni/nni/compression/torch/pruning/sensitivity_pruner.py

```diff
@@ -383,14 +394,20 @@ def compress(self, eval_args=None, eval_kwargs=None,
                 with open(cfg_path, 'w') as jf:
                     json.dump(cfg_list, jf)
                 self.analyzer.export(sensitivity_path)
+
             if cur_ratio > target_ratio:
                 # If this is the last prune iteration, skip the time-consuming
                 # sensitivity analysis
+
                 self.analyzer.load_state_dict(self.model.state_dict())
                 self.sensitivities = self.analyzer.analysis(
                     val_args=eval_args, val_kwargs=eval_kwargs)
 
         _logger.info('After Pruning: %.2f weights remains', cur_ratio)
+        self.modules_wrapper = modules_wrapper_final
+        # TODO double check if return a normal model or
+        # a wrapped model
```

Comment:
could you check the model and remove this TODO?

Reply:
Checked and updated.